src/activity_handlers.cpp

```diff
@@ -2067,14 +2068,15 @@ repeat_type repeat_menu( const std::string &title, repeat_type last_selection )
     rmenu.text = title;
 
     rmenu.addentry( REPEAT_ONCE, true, '1', _( "Repeat once" ) );
-    rmenu.addentry( REPEAT_FOREVER, true, '2', _( "Repeat as long as you can" ) );
+    rmenu.addentry( REPEAT_FOREVER, true, '2', _( "Repeat until reinforced" ) );
```

Comment:
Is this text appearing for items that are not reinforced (like weapons, which are accurized?).  That may be misleading (don't know if it's a big problem, though).

Reply:
Interestingly, this repair_item activity only applies for some tools (all the tailoring related tools, soldering iron and the various welders). AFAIK the term "accurized" only applies to gun repair which is not covered here. Melee weapons still use the "reinforced" description. In fact the current title in the popup window will say "reinforcing", so this term here should be valid.